rust/Makefile

```
@@ -110,10 +111,11 @@ rustdoc-alloc: $(src)/alloc/lib.rs rustdoc-core rustdoc-compiler_builtins FORCE
 	$(call if_changed,rustdoc)
 
 rustdoc-kernel: private rustc_target_flags = --extern alloc \
-    --extern build_error --extern macros=$(objtree)/$(obj)/libmacros.so
+    --extern build_error --extern macros=$(objtree)/$(obj)/libmacros.so \
+	--extern bindings
```

We indent these with 4 spaces to distinguish it from the recipe prefix character:

```suggestion
    --extern bindings
```